po/ja.po

```diff
@@ -0,0 +1,108 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.artemanufrij.imageburner package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.artemanufrij.imageburner\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-09-22 18:40+0200\n"
+"PO-Revision-Date: 2018-10-27 12:43+0100\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.2\n"
+"Last-Translator: Ciarán Ainsworth <ciaranainsworth@outlook.com>"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Language: ja\n"
+
+#: src/MainWindow.vala:67 src/MainWindow.vala:89
+msgid "Change"
+msgstr "変更"
+
+#: src/MainWindow.vala:70 src/MainWindow.vala:335
+msgid "Open"
+msgstr "開く"
+
+#: src/MainWindow.vala:97 src/MainWindow.vala:256
+msgid "Device"
+msgstr "デバイス"
+
+#: src/MainWindow.vala:124
+msgid "Calculating checksum…"
+msgstr "チェックサムの計算中…"
+
+#: src/MainWindow.vala:144
+msgid "please wait…"
+msgstr "お待ちください…"
+
+#: src/MainWindow.vala:156 src/MainWindow.vala:422
+msgid "Ready!"
+msgstr "準備完了!"
+
+#: src/MainWindow.vala:158
+#, c-format
+msgid "%s was written onto %s"
+msgstr "%s は %s に書き込まれました"
+
+#: src/MainWindow.vala:185
+msgid "Image Burner"
+msgstr "イメージバーナー"
+
+#: src/MainWindow.vala:194
+msgid "Choose an algorithm"
+msgstr "アルゴリズムを選択する"
+
+#: src/MainWindow.vala:211
+msgid "Finished"
+msgstr "完成した"
```

Comment:
I think we should use "完了" because this label is to tell users the process writing to drive is completed.
https://hinative.com/ja/questions/27554
http://lang-8.com/303464/journals/1201535

```suggestion
msgstr "完了しました"
```